persistence/sql/persister_identity.go

```diff
@@ -559,66 +554,28 @@ func (p *Persister) searchCredentialQuery(field string, values []string) pop.Sco
 	}
 }
 
-var quoteChar = map[string]uint8{
-	"cockroach": '"',
-	"mariadb":   '`',
-	"mysql":     '`',
-	"postgres":  '"',
-	"sqlite3":   '"',
-}
-
-func (p *Persister) Quote(ctx context.Context, key string) string {
-	n := p.Connection(ctx).Dialect.Name()
-	c, ok := quoteChar[n]
-	if !ok {
-		// guess panic is OK here as the error is not fixable without a new release of Kratos
-		panic("DSN is of unknown dialect " + n)
-	}
-
-	parts := strings.Split(key, ".")
-
-	for i, part := range parts {
-		part = strings.Trim(part, `"`)
-		part = strings.Trim(part, "`")
-		part = strings.TrimSpace(part)
-
-		parts[i] = fmt.Sprintf(`%c%v%c`, c, part, c)
-	}
-
-	return strings.Join(parts, ".")
-}
```

Comment:
I wasn't sure where this code comes from. Looking at it, I am not very confident that it actually prevents SQL injection. What about other escape characters such as `'` or `--` or other unicode chars?